pycsob/utils.py

```diff
@@ -5,23 +5,15 @@
 from Crypto.Hash import SHA
 from Crypto.PublicKey import RSA
 from Crypto.Signature import PKCS1_v1_5
+from urllib.parse import urljoin, quote_plus
 
 from . import conf
 
 
 try:
-    from django.utils.timezone import localtime as now
+    from django.utils import timezone as datetime
 except ImportError:
-    from datetime import now
```

Comment:
Je to `datetime.datetime.now`, jak se zdá. Ale `datetime.datetime` není package, takže to nejde importovat jako `from datetime.datetime import now`

Reply:
I've encountered this as well, strange, is it because of new Python version? I believe it was possible in past.

Reply:
Not sure anymore but I don't think so - if I remember correctly, it failed for all 3 python version. Maybe worked for Py2?

Reply:
from datetime import now was nonsence

Reply:
ando from datetime.datetime import now was never posible